test/controllers/api/v2/subscriptions_controller_test.rb

```diff
@@ -63,6 +64,13 @@ def test_index_protected
       end
     end
 
+    def test_index_with_activation_key_id
+      activation_key = katello_activation_keys(:simple_key)
+      get :index, activation_key_id: activation_key.id
+      assert_response :success
+      assert_template 'api/v2/subscriptions/index'
```

Comment:
@johnpmitsch Is there an example in the controller tests for testing proper output?